product_docs/docs/eprs/6.2/03_installation/01_installing_with_stackbuilder.mdx

```diff
@@ -193,3 +193,10 @@ An information box appears showing the installation progress of the xDB Replicat
 **Figure 3-25: StackBuilder Plus installation complete**
 
 Successful installation of xDB Replication Server results in the creation of directory structures and files in your host environment as described in Section [Post-Installation Host Environment](05_post_installation_host_environment/#post_installation_host_environment).
+
+**Step 22:** Install a JDBC driver. See [Installing a JDBC driver](installing_jdbc_driver) for more information.
+
+!!! Note
```

Comment:
Is this note applicable on Windows?  It doesn't appear in the EPRS 7 Windows/stackbuilder installation page. 

If it is applicable, I think it would be better to include at the end of Step 1 as that is where the JDK installation is discussed.